models/callbackalert.go

```diff
@@ -39,6 +40,9 @@ func (ca CallbackAlert) MarshalJSON() ([]byte, error) {
 	return json.Marshal(test)
 }
 
+/*
+ * String function for representing a CallbackAlert
```

Comment:
What was the rationale for using a block comment here?

Reply:
On all types String() has a block comment so I made it the same here rather than changing them all